prime-router/src/main/kotlin/cli/LookupTableCommands.kt

```diff
@@ -528,11 +536,20 @@ class LookupTableCreateCommand : GenericLookupTableCommand(
     private val tableName by option("-n", "--name", help = "The name of the table to perform the operation on")
         .required()
 
+    /**
+     * Force to create table(s).
+     */
+    private val force by option("-f", "--force", help = "Force to create the table").flag()
+
     override fun run() {
         // Read the input file.
         val inputData = csvReader().readAllWithHeader(inputFile)
-        if (inputData.size <= 1)
-            error("Input file ${inputFile.absolutePath} has no data.")
+        // Note: csvReader returns size of data-row(s) and NOT include the header-row.
+        // (i.e. If the file contains of header row, it returns size = 0)
+        if (inputData.size < 1) {
+            TermUi.echo("WARNING: Input file ${inputFile.absolutePath} has no data.")
```

Comment:
If you return then this is not a warning, but an error

Reply:
Done.